rectools/metrics/popularity.py

```diff
@@ -32,17 +34,23 @@ class AvgRecPopularity(MetricAtK):
     with this item.
 
     .. math::
-        ARP@k = \frac{1}{\left|U_{t}\right|}\sum_{u\in U_{t}^{}}\frac{\sum_{i\in L_{u}}\phi (i)}{\left| L_{u} \right |}
+        ARP@k = \frac{1}{|U_{t}|}\sum_{u\in U_{t}^{}}\frac{\sum_{i\in L_{u}}\phi (i)}{|L_{u}|}
+    .. math::
+        Normalized ARP@k = \frac{1}{|U_{t}|}\sum_{u\in U_{t}^{}}\frac{\sum_{i\in L_{u}}\frac{\phi(i)}{|I|}}{|L_{u}|}
 
     where
```

Comment:
You need to add indents here. This does not show in the docs correctly: https://rectools.readthedocs.io/en/latest/api/rectools.metrics.popularity.AvgRecPopularity.html#rectools.metrics.popularity.AvgRecPopularity
See `Serendipity` metric for example